apps/frontend/src/app/5_pages/BitocracyPage/components/NewProposalForm/components/ProposalDataForm/ProposalDataForm.constants.ts

```diff
@@ -0,0 +1,3 @@
+export const maxCharactersLength = 140;
```

Comment:
Split it into 2 constants - `MAXIMUM_TITLE_LENGTH` and `MAXIMUM_SUMMARY_LENGTH`.